# Calculate format=diff pkg(sys-kernel/calculate-sources)=>5.10,5.11&&merge(x11-drivers/nvidia-drivers[-uvm])=>390,397
diff --git a/kernel/Makefile b/kernel/Makefile
index 23f630c..e354b74 100644
--- a/kernel/Makefile
+++ b/kernel/Makefile
@@ -59,7 +59,7 @@ else
     )
   endif
 
-  NV_KERNEL_MODULES ?= $(wildcard nvidia nvidia-uvm nvidia-modeset nvidia-drm nvidia-vgpu-vfio)
+  NV_KERNEL_MODULES ?= $(wildcard nvidia nvidia-modeset nvidia-drm nvidia-vgpu-vfio)
   NV_KERNEL_MODULES := $(filter-out $(NV_EXCLUDE_KERNEL_MODULES), \
                                     $(NV_KERNEL_MODULES))
   NV_VERBOSE ?=
